Automattic/vip-go-mu-plugins
# Search: switch vip-search index --network-wide to spawn child processes instead of using switch_to_blog

## Description

The current implementation may cause the code not to be loaded - `switch_to_blog` doesn't do that, leading to issues like post types missing from the index for a specific site. 

E.g. if a subsite has a plugin that the main site is not loading, a post type registered by that plugin will be missing from the index. 

This PR addresses the issue by utilizing `WP_CLI::runcommand` which spawns a child process and switches execution to it, thus ensuring the proper codebase is loaded.

We will likely need to extend the approach to other `--network-wide` commands.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out PR.
1. Run `wp vip-search index --network-wide` on multisite
1. Verify that indexing runs through without issues
1. Run `wp vip-search index` and verify that only the first site is indexed
